src/hrc721.spec.ts

```diff
@@ -71,4 +71,18 @@ describe('HarmonyProvider', () => {
       expect(provider.ownerOf('')).to.be.rejectedWith(Error)
     })
   })
+
+  describe('getApproved', () => {
+    it('should returns the account approved for tokenId token', async () => {
+      const approved = await provider.getApproved('1')
+      expect(approved).to.exist
+      expect(approved).to.not.be.null
+      expect(approved).to.not.be.undefined
+      expect(approved).to.be.equals('0x0000000000000000000000000000000000000000')
+    })
+
+    it('should throw an error if tokenId is not provided', async () => {
+      expect(provider.getApproved('')).to.be.rejectedWith(Error)
+    })
+  })
```

Comment:
let's leave a TODO here so that we can add more tests when we have the approve function working

Reply:
Done!